src/main/java/com/employeesService/EmployeesService/model/Department.java

```diff
@@ -20,4 +38,23 @@ public String getDeptName() {
     public void setDeptName(String deptName) {
         this.deptName = deptName;
     }
+
+    public void addEmployee(Employee employee, LocalDate fromDate) {
+        DepartmentEmployee employeeToAdd = new DepartmentEmployee(this, employee);
+        employeeToAdd.setFromDate(fromDate);
+        employees.add(employeeToAdd);
+    }
+
+    public void deleteEmployee(Long empNo, LocalDate toDate) {
+        DepartmentEmployeeId id = new DepartmentEmployeeId(deptNo, empNo);
+        this.employees.stream()
+                .filter(employee -> employee.getId().equals(id))
+                .findAny()
+                .orElseThrow(() -> new DepartmentNotFoundException(deptNo))
```

Comment:
You're not looking up for Department, but for Employee, so `EmployeNotFoundException(empNo)` would be more appropriate